service-commands/src/index.js

```diff
@@ -1,67 +1,22 @@
-const {
-  runSetupCommand,
-  performHealthCheck,
-  allUp,
-  Service,
-  SetupCommand
-} = require('./setup')
+const Setup = require('./setup')
 
 const { LibsWrapper, Utils } = require('./libs')
 
-// TODO: should make all of these imports dynamic!!!
-const {
-  addUser,
-  upgradeToCreator,
-  autoSelectCreatorNodes,
-  getUser,
-  getUserAccount,
-  getLibsWalletAddress,
-  setCurrentUser,
-  getLibsUserInfo,
-  updateMultihash,
-  updateCoverPhoto,
-  updateProfilePhoto
-} = require('./commands/users')
-const {
-  uploadTrack,
-  getTrackMetadata,
-  addTrackToChain,
-  updateTrackOnChain
-} = require('./commands/tracks')
-const { verifyCIDExistsOnCreatorNode } = require('./commands/files')
-const { addIPLDToBlacklist } = require('./commands/ipldBlacklist')
-const {
-  createPlaylist,
-  updatePlaylistCoverPhoto,
-  getPlaylists
-} = require('./commands/playlists')
+// Any method you add in these commands files will be dynamically imported
```

Comment:
Nit: let's not refer to this as `dynamic import` since that already means a specific thing in Javascript Land!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Dynamic_Imports